lib/pf/mfa/Akamai.pm

```diff
@@ -410,6 +411,7 @@ sub _get_curl {
     $curl->setopt(CURLOPT_SSL_VERIFYPEER, 0);
     $curl->setopt(CURLOPT_HEADER, 0);
     $curl->setopt(CURLOPT_WRITEDATA,$fileb);
+    $curl->setopt(CURLOPT_TIMEOUT_MS, 300)
```

Comment:
Just to make sure, when we're waiting for the push to complete, if we timeout after 300ms and retry lets say 15 times for 300ms, when the user will have accepted the push, the 16th time the request will get accepted

Reply:
In fact we just check the status if the user accept the push notification on his device.
So it´s here, the maximum time to have the reply but it´s in a loop.
https://github.com/inverse-inc/packetfence/blob/feature/MFA/lib/pf/mfa/Akamai.pm#L281